doc/guides/conngen_interface.rst

```diff
@@ -6,10 +6,9 @@ Since 2014 (`Djurfeldt et al.,
 Connection Generator Interface. This allows to couple connection
 generating libraries to NEST without having to modify NEST itself.
 
-In contrast to the `built-in connection
-functions :doc:`Connection management <connection_management>` and the Topology
-module, the Connection Generator Interface has a different way for specifying
-the connectivity:
+In contrast to the :doc:`Connection management <connection_management>` and
+Topology module's built-in connection functions, the Connection Generator
```

Comment:
Do we still talk about Topology? Isn't this now part of regular `Connect` somehow? Or should this also include a link to the corresponding manual? Sorry if I missed something.